mmv1/third_party/terraform/services/workstations/resource_workstations_workstation_test.go.erb

```diff
@@ -137,6 +141,10 @@ resource "google_workstations_workstation" "default" {
   labels = {
 	  foo = "bar"
   }
+
+  env = {
+    name = "bar"
```

Comment:
Can the value be updated to a different value to see if the updating of this field works?

Reply:
@zli82016 done!

Reply:
Thanks.